src/commands/server/subscribe.ts

```diff
@@ -16,40 +17,39 @@ export default class Subscribe {
     this.db = db;
     this.data = new SlashCommandSubcommandBuilder()
       .setName("subscribe")
-      .setDescription("Subscribe to the news of the current server you're in.");
+      .setDescription("Subscribe to the news of this server.");
   }
 
   async run(interaction: ChatInputCommandInteraction) {
     const db = this.db;
-    const newsTable = await getNewsTable(db);;
-
+    const newsTable = await getNewsTable(db);
     const guild = interaction.guild;
     const user = interaction.user;
 
-    let subscriptions = await newsTable?.get(`${guild.id}.subscriptions`).then(
-      (subscriptions: any) => subscriptions as SubscriptionType ?? [] as SubscriptionType
-    )
+    let subscriptions = await newsTable
+      ?.get(`${guild.id}.subscriptions`)
```

Comment:
```suggestion
    let subscriptions = await newsTable?
      .get(`${guild.id}.subscriptions`)
```
That character belongs there I think ;)